bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/common/SessionController.java

```diff
@@ -46,11 +46,10 @@
    @Inject
    private ModelRepository modelRepository;
 
-   private final Codecs encoder;
+   @Inject
+   private Codecs encoder;
 
-   public SessionController() {
-      this.encoder = new Codecs();
-   }
+   public SessionController() {}
```

Comment:
I believe we can remove the constructor here if it doesn't do anything of value.